src/diem/jsonrpc/client.py

```diff
@@ -567,8 +570,10 @@ def _send_http_request(
         request: typing.Dict[str, typing.Any],
         ignore_stale_response: bool,
     ) -> typing.Dict[str, typing.Any]:
+        self._logger.debug("http request body: %s", request)
         response = self._session.post(url, json=request, timeout=self._timeout)
         response.raise_for_status()
+        self._logger.debug("http response body: %s", response.text)
```

Comment:
maybe put this above the `raise_for_status` in case there is additional helpful info in the body?